modules/cnf-topology-aware-lifecycle-manager-cluster-logging-6x-update-policygenerator.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="talm-cluster-logging-6x-update-policygenerator_{context}"]
+= Removing Cluster Logging Operator artifacts with {policy-gen-cr} CRs
+
+When you update to Cluster Logging Operator version 6.x from version 5.x, you must add a cleanup policy that removes the old Operator artifacts from deployed clusters after creating the new API custom resources (CRs).
+
+.Procedure
+
+. Add the `acm-group-du-clo5-cleanup.yaml` file in the git repository that contains your other `PolicyGenerator` CRs.
```

Comment:
> Add the `acm-group-du-clo5-cleanup.yaml` file in the git repository

This line confuses me. Add the file _to_ the repository? Or am I adding something to the file that is already in the repo? 
Is the example below what you use to create the file? If so, maybe make the name of the YAML and the `name:` value in the code the same? Is that possible?

Reply:
Updated it `to the repository`. I won't change the file name and the name value because this is how you get the file from the ztp container image: https://github.com/openshift-kni/telco-reference/blob/main/telco-ran/configuration/argocd/example/acmpolicygenerator/acm-group-du-clo5-cleanup.yaml